development/webpack/webpack.config.ts

```diff
@@ -57,6 +55,7 @@ if (args.manifest_version === 3) {
 // #endregion short circuit for unsupported build configurations
 
 const context = join(__dirname, '../../app');
+const projectRoot = join(__dirname, '../../'); // While ../../app is the main dir for the webpack build to use as context, the project root where package.json is one level up. This discrepancy needs to be explained to LavaMoat plugin as it's searching for the package.json in the compilator.context by default.
```

Comment:
`process.env.npm_package_version` is an env var set by yarn and npm that lavamoat can check first, before looking for the `package.json`.

But also, we could also look into changing `context` to use the project root, as it's caused problems a few times now.

Reply:
No, not really. The lavamoat plugin should probably implement npm's "package root resolution" algorithm in order to find the `package.json` file, but this is also fine.

Reply:
Webpack's default behavior is to rely on compiler `context` which is a path to the project root. If LavaMoat needs to treat something else as the project root for the sake of exploring the dependencies and such, there's the option to point to it, but by default we need to follow the webpack's convention. 
AFAIR the only reason the root setting is needed here is that our build explicitly sets context.
Theoretically, if package.son is not found in the context and root is not set, we could keep going up until we find a package.json but there's a risk it'd be an unpleasant surprise for some projects. Happy to be convinced otherwise.